src/protocols/tendermint/contract.js

```diff
@@ -0,0 +1,12 @@
+module.exports = class TendermintContract {
+  static identifierName() {
+    return 'address'
```

Comment:
Hmmm perhaps we should change this, from my understanding we'll have "chains" instead of "contracts".